Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -139,7 +139,7 @@ public extension UIImage {
                                      width: destRect.width.rounded(),
                                      height: destRect.height.rounded())
 
-        UIGraphicsBeginImageContext(roundedDestRect.size)
+        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, false, scale)
```

Comment:
Just to make it easier for the reader ... I had to go to docs to remember what `false` meant 
```suggestion
        UIGraphicsBeginImageContextWithOptions(roundedDestRect.size, /*opaque=*/false, scale)
```